projects/addon-editor/components/toolbar-tools/details-remove/details-remove.component.ts

```diff
@@ -0,0 +1,29 @@
+import {ChangeDetectionStrategy, Component, Inject} from '@angular/core';
+import {TuiEditor} from '@taiga-ui/addon-editor/abstract';
+import {TuiTiptapEditorService} from '@taiga-ui/addon-editor/directives';
+import {TUI_EDITOR_TOOLBAR_TEXTS} from '@taiga-ui/addon-editor/tokens';
+import {LanguageEditor} from '@taiga-ui/i18n';
+import {Observable} from 'rxjs';
+import {distinctUntilChanged, map} from 'rxjs/operators';
+
+@Component({
+    selector: 'tui-details-remove',
+    templateUrl: './details-remove.template.html',
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class TuiDetailsRemoveComponent {
+    readonly disabled$ = this.editor.stateChange$.pipe(
```

Comment:
Does this repeat for every action? Maybe we can add a token for action name and move this to abstract class? Maybe even `removeDetails` can be something like `performAction` and by that token it would figure out what to call from `editor`?